modules/light-clients/08-wasm/types/keys.go

```diff
@@ -6,4 +6,7 @@ const (
 
 	// StoreKey is the store key string for 08-wasm
 	StoreKey = ModuleName
+
+	// Wasm is used to indicate that the light client is a on-chain wasm program
+	Wasm string = ModuleName
```

Comment:
do we need the `string` beside the const name? 

suggestion for godoc:

```suggestion
	// Wasm is the client type for IBC light clients created using 08-wasm
	Wasm string = ModuleName
```